rust/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 0.13.0
+
+* Updates from mediasoup TypeScript `3.13.0..=3.13.7`.
```

Comment:
I think we should highlight transition to flatbuffers and io_uring support here

Reply:
Done here: https://github.com/versatica/mediasoup/pull/1247/commits/73a3a6d5841d24a2633cf418dce60eade25ee1dc